### ISSUE TITLE
Is "Select your country" the best micro copy?

### ISSUE THREAD

**Issue by stephenkeable-allies** [alphagov/accessible-autocomplete#161]
Minor thought about the label for the fields on the examples, which is currently "Select your country" above a text input field. So seemingly with nothing to "select".
For screen readers the status element would advise the user needs to start typing, however other users don't see or hear that text instruction.
Wonder if it's a good idea to use examples with label of "what's your country" and a hint of "Start typing your country" or similar, to encourage developers to use sensible labels too.
of course you'd like to think people are considering this anyway, but some may copy and paste the element and labels without much thought.

**Comment by joelanman** [comment]
good point - should probably just be "Country" for the purposes of the examples.

In a real service we'd want the copy to fit the context. So on a page that asked "What is your addess?" a fields with "Postcode", "Country", etc might be fine. In another context the question might be something like "What country were you born in?" which is a bit wordy and specific for a page of multiple examples.

**Comment by stephenkeable-allies** [comment]
Main issue was the word "select" when it's an element you "type" in.
Have you guys done any research on whether the user needs to be told that results will appear as they start typing, before they interact with the field?
I've seen cases where users who look at the keyboard while typing, end up typing the entire name, not knowing they could have selected the item they wanted after a few keystrokes.

**Comment by edwardhorsford** [comment]
This isn't the language we were using in user research. We had context appropriate questions.

@stephenkeable-allies We've not tried explicitly telling the users that results will appear, but have seen no issues with users inputting country information. We're asking for a piece of information they know, so as long as they know a name for it, it works well for them.

We'll blog a bit about what we found in the research after Purdah. Broadly users fell in to two categories: those that typed two or three characters and then picked a country from suggestions, or those that typed the entire first word of a country. Rarely did users type a full country name containing two or more words. 

As long as they're able to get to a country , we don't much mind how they use it - there's no reason to push them towards 'searching' - as they know the name of the thing. For contexts where the user might not be as confident, I'd agree it would be good for the text to give a hint that you can search.

**Comment by stephenkeable-allies** [comment]
@edwardhorsford very useful as ever, thanks. Having open government departments researching and sharing knowledge is incredibly useful. Look forward to the blog about the research, sounds like users must look up after the first word, which is interesting.

**Comment by tvararu** [comment]
Hello! 👋 

I wrote the content of the `<label>` and can confirm that no thought whatsoever went into it! The docs need a lot of work. The crude examples were hastily put together to demonstrate the API and provide a playground for manual and automated testing.

This is great and please feel free to quickly submit a PR redesigning the content. 👍
